packages/sparrowql-graphql/package-lock.json

```diff
@@ -16,6 +16,9 @@
 			"version": "1.2.2",
 			"resolved": "https://registry.npmjs.org/iterall/-/iterall-1.2.2.tgz",
 			"integrity": "sha512-yynBb1g+RFUPY64fTrFv7nsjRrENBQJaX2UL+2Szc9REFrSNm1rpSXHGzhmAy7a9uv3vlvgBlXnf9RqmPH1/DA=="
+		},
+		"sparrowql": {
+			"version": "file:../sparrowql"
```

Comment:
I don't think this is correct. Could you reset this file and check if Lerna works correctly without this change?